### PR TITLE
Feature File for Story AV-86

### PR DESCRIPTION
**Pull request by Abhishek8694**
Hi Abhilasha Wagh,
Please review the feature file for story **"AV- 86"**.
and let me know the changes if any.

Thanks,
Abhishek Vaidya

### CODE REVIEW

File: src/test/resources/Features/productDescription.feature [hunk 1]
```diff
@@ -0,0 +1,35 @@
+
+@tag
+Feature: Title of your feature
+  I want to use this template for my feature file
+
+  @tag1
+  Scenario: Title of y*ur $cen@r!o
```
Comment (abhilashawagh) [inline src/test/resources/Features/productDescription.feature]: Remove symbols at scenario line

File: src/test/resources/Features/productDescription.feature [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    Given I want to write a step with precondition
+    And some other precondition
+    When I complete action
+    And some other action
+    And yet another action
+    Then I validate the outcomes
+    And check more outcomes
+    
+     @tag2*7/2
```
Comment (abhilashawagh) [inline src/test/resources/Features/productDescription.feature]: delete *7/2 at tag line

**Review by abhilashawagh — COMMENTED** [review]
Please make associated changes in feature file for AV-86 story

File: src/test/resources/Features/productDescription.feature (hunk 1 above)
Comment (Abhishek8694) [reply src/test/resources/Features/productDescription.feature]: made the changes

File: src/test/resources/Features/productDescription.feature (hunk 2 above)
Comment (Abhishek8694) [reply src/test/resources/Features/productDescription.feature]: made the changes

**Comment by abhilashawagh** [comment]
You can merge the branch to master.
Thanku.

**Pull request merged**